src/cryptonote_basic/miner.h

```diff
@@ -30,7 +30,6 @@
 
 #pragma once 
 
-#include <boost/program_options.hpp>
 #include <boost/logic/tribool_fwd.hpp>
```

Comment:
There's still two references in this header but no obvious forward declare ... ? Indirectly via `cryptnote_basic.h` (or elsewhere) is kind of dodgy.

Reply:
Yes, it was referenced via cryptonote_basic.h, as I learned from Doxygen dep. graph. Now it's included directly.